src/components/previewCard/imageHelper.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+
+interface Props {
+  imageSrc?: string
+  pictureAlt?: string
```

Comment:
We should render this component only if we have the imageSrc, so `imageSrc` and `pictureAlt` should be mandatory